modules/xmpp/JingleSessionPC.js

```diff
@@ -339,31 +339,41 @@ JingleSessionPC.prototype.sendAnswer = function (success, failure) {
 
 JingleSessionPC.prototype.createdAnswer = function (sdp, success, failure) {
     //logger.log('createAnswer callback');
```

Comment:
Thank you, @iloire! +1 from me on deleting unused comments in general. In the case, I'd rather not act on this comment of yours because:

- There are multiple "unused comments" throughout this and other files. If we are to remove them, I'd prefer to have that done in a separate dedicated PR.

- I'm not even 100% sure that this particular comment is truly unused. I guess there's a chance it was left for easy uncommenting.